client/e2e/pages/EventsPage.ts

```diff
@@ -157,25 +162,54 @@ export class EventsPage {
   "123",
   "http://"
   */
-  async addnewevent(event: EventInfo) {
+  async addnewevent(anEvent: EventInfo) {
     await this.addButton.click();
     await this.eventNameBlank.click();
-    await this.page.getByLabel('Event Name:').fill(event.eventName);
+    await this.page.getByLabel('Event Name:').fill(anEvent.eventName);
     await this.eventDesBlank.click();
     await this.page
       .getByLabel('Event Description:')
-      .fill(event.eventDescription);
+      .fill(anEvent.eventDescription);
     await this.imageURL.click();
-    await this.page.getByLabel('Image URL:').fill(event.eventURL);
+    await this.page.getByLabel('Image URL:').fill(anEvent.eventURL);
     await this.newEventSave.click();
     await this.eventContinue.click();
   }
+  async activeEvent()
+  {
+     await this.activateEventchecker.check();
+  }
 
-  /**
+  /**----------------------------------------------
    * Switches the status of the event's activeness.
    */
+  /*
   async activateEvent() {
     await this.eventOption2.click();
+  }-------------------------------------------------*/
+  async addNewInactiveEvent(anEvent: EventInfo)

```

Comment:
get rid of this. just use add event, this is the exact same as add event. it is duplicate code

Reply:
like i mentioned in another comment. I would suggest keeping adding event and activating event separate.